bundles/org.openhab.binding.solarman/README.md

```diff
@@ -0,0 +1,305 @@
+# Solarman Logger Binding
+
+[![Build Status](https://app.travis-ci.com/catalinsanda/org.openhab.binding.solarman.svg?branch=main)](https://app.travis-ci.com/catalinsanda/org.openhab.binding.solarman)
+
+Binding used to communicate with Solarman (IGEN-Tech) v5 based solar inverter data loggers in direct-mode over the local
+network.
+More information about the different types of stick loggers is available on the
+[Solarman](https://www.solarmanpv.com/products/data-logger/stick-logger/) site.
+
+These data loggers are used by inverters from a lot of manufacturers, just to name a few:
+
+- Deye
+- Sofar
+- Solis
+- ZCS Azzurro
+- KStar
+
+## Supported Things
+
+The `solarman:logger` thing supports reading data from a Solarman LSW-3 Stick Logger (it might also work with LSE-3 and
+maybe others)
+when connected to a supported inverter.
+
+It was tested on a SUN-10K-SG04LP3-EU only, but because the implementation uses the inverter definitions created as part
+of
+[Stephan Joubert's Home Assistant plugin](https://github.com/StephanJoubert/home_assistant_solarman)
+it **might** work with the other inverters supported by the plugin.
+
+## Thing Configuration
+
+To connect the logger you need the IP address of the logger and its serial number. The IP address can be obtained from
+your router and the serial number can either be read from the label of the logger, or by connecting to the logger with
+a browser (default user/pass: admin/admin) and getting it from the Status page. Please note that you need the
+"Device serial number" from the "Device information" section, not the "Inverter serial number".
+
+### `solarman:logger` Thing Configuration
```

Comment:
```suggestion
### `logger` Thing Configuration
```

Not wrong, but usually the readme does not have the bindingid listed in the uid.